iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -157,9 +166,12 @@ public class HiveIcebergStorageHandler implements HiveStoragePredicateHandler, H
   private static final String TABLE_NAME_SEPARATOR = "..";
   private static final String ICEBERG = "iceberg";
```

Comment:
we can remove this and use the Constants

Reply:
Done, moved it to Constants